packages/embeds/embed-core/src/preview.ts

```diff
@@ -47,7 +47,7 @@ if (!bookerUrl || !embedLibUrl) {
       }
       p(cal, ar);
     };
-})(window, "//localhost:3000/embed/embed.js", "init");
+})(window, embedLibUrl, "init");
```

Comment:
It was an accidental change that got committed.
https://github.com/calcom/cal.com/pull/13386/files#r1586013449